src/arrayMethodReduce.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let totalValue = initialValue;
```

Comment:
usually `total` is used with numbers - `finalResult`can be used instead or something else